### PR TITLE
Add Pastebin.ca pastebin site

### PR DESCRIPTION
**Pull request by febrezo**
Added a new translator for the paste site pastebin.ca. It improves default metadata extraction by adding website type, the paste's publication date and paste's title.

### CODE REVIEW

File: Pastebin.ca.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+	newItem.attachments.push({
+		title: "Pastebin.ca Snapshot",
+		mimeType: "text/html",
+		url: url
+	});	
+	
+	newItem.complete();
+}
+/** BEGIN TEST CASES **/
+var testCases = [
```
Comment (adam3smith) [inline Pastebin.ca.js]: Same here -- could we get a non-spammy test?

File: Pastebin.ca.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
+{
+	"translatorID": "bce84d9c-3e2b-45c6-b3b4-5176c00badd6",
+	"label": "Pastebin.ca",
+	"creator": "febrezo",
+	"target": "^https?://pastebin.ca/[0-9]+",
```
Comment (adam3smith) [inline Pastebin.ca.js]: Escape periods in regex (`...pastebin\.com`)

File: Pastebin.ca.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+	// Creating the item
+	var newItem = new Zotero.Item(resourceType);
+
+	//Setting common data:
+	newItem.websiteTitle = "Pastebin.ca";
+	newItem.websiteType = "Paste Site";
+	newItem.url = url;
+	var title = ZU.xpathText(doc, '//h2/dl/dt');	
+	newItem.title = title;
+	//var date = ZU.xpathText(doc, "//div[@id='pasteInfo']/div/span[text()='Pasted:']/following-sibling::text()[1]");
```
Comment (adam3smith) [inline Pastebin.ca.js]: that seems of little use as a comment. Delete.

**Comment by AbeJellinek** [comment]
Pastebin.ca has been [dead since mid-2018](https://web.archive.org/web/20180812110936/https://pastebin.ca/), so I'm closing this.

**Pull request closed without merging**